content/hardware/06.nicla/boards/nicla-voice/tutorials/user-manual/content.md

```diff
@@ -439,7 +439,262 @@ The onboard magnetometer of the Nicla Voice can be used to determine the board's
+  y_acc_raw = (0x0000 | sensor_data[2] | sensor_data[3] << 8);
+  z_acc_raw = (0x0000 | sensor_data[4] | sensor_data[5] << 8);
+  x_gyr_raw = (0x0000 | sensor_data[6] | sensor_data[7] << 8);
+  y_gyr_raw = (0x0000 | sensor_data[8] | sensor_data[9] << 8);
+  z_gyr_raw = (0x0000 | sensor_data[10] | sensor_data[11] << 8);
+
+  // Convert raw accelerometer data to acceleration expressed in meters per second squared. 
+  x_acc = x_acc_raw * ACCEL_SCALE_FACTOR;
+  y_acc = y_acc_raw * ACCEL_SCALE_FACTOR;
+  z_acc = z_acc_raw * ACCEL_SCALE_FACTOR;
+
+  // Convert raw gyroscope data to angular velocity expressed in degrees per second.
+  x_gyr = x_gyr_raw * GYRO_SCALE_FACTOR;
+  y_gyr = y_gyr_raw * GYRO_SCALE_FACTOR;
+  z_gyr = z_gyr_raw * GYRO_SCALE_FACTOR;
+  
+  // Print accelerometer data (expressed in meters per second squared). 
+  Serial.print("x_acc:");
+  Serial.print(x_acc);
+  Serial.print(",");
+  Serial.print("y_acc:");
+  Serial.print(y_acc);
+  Serial.print(",");
+  Serial.print("z_acc:");
+  Serial.println(z_acc);
+
+  // Print gyroscope data (expressed in degrees per second). 
+  Serial.print("x_gyr:");
+  Serial.print(x_gyr);
+  Serial.print(",");
+  Serial.print("y_gyr:");
+  Serial.print(y_gyr);
+  Serial.print(",");
+  Serial.print("z_gyr:");
+  Serial.println(z_gyr);
+
+  delay(1000);
+}
+```
+
+Here you can find a step-by-step explanation of the code:
+
+First, the necessary libraries are included: 
+
+- `NDP.h` and `BMI270_Init.h` for the Nicla Voice board's basic functions and the IMU control.
+- Macros are defined for checking the status of the IMU; these macros allow the sketch to detect and handle sensor errors. 
+
+Next, user functions `ledBlueOn()`, `ledGreenOn()`, and `ledRedBlink()` definition: 
+
+- These functions allow the onboard LEDs to flash specific colors to indicate different states: blue for a successful match, green for an event, and red to indicate an error.
+
+Next, in the `setup()` function:
+
+- The serial communication is initialized at a baud rate of 115200.
+- The Nicla Voice board is initialized, and the LDO is disabled.
+- Error and event handlers are initialized.
+- NDP processor is initialized; this process includes populating the external Flash memory of the board with the NDP processor internal microcontroller firmware (`mcu_fw_120_v91.synpkg`), the NDP processor internal DSP firmware (`dsp_firmware_v91.synpkg`), and the ML model (`ei_model.synpkg`). 
+- The BMI270 sensor is initialized; this includes a software reset, loading the sensor configuration, and setting it into normal power mode with the accelerometer and gyroscope operational. 
+
+Finally, in the `loop()` function:
+
+- Memory is allocated for the sensor data; data is then read from the sensor and stored in this allocated space.
+- Raw sensor data is then parsed and extracted into raw accelerometer and gyroscope data. This data is represented as 16-bit signed integers ranging from -32,768 to 32,767.

```

Comment:
I have some fears here:

32,767 it s 32767 (32 thousand bla bla bla)

Should we put the USA comma here (European point) @jhansson-ard or without it to avoid confusion across different users in different parts of the world?

Reply:
1. 32,767 (thirty-two thousand seven hundred and seven)
2. 32.676 (thirty-two point six seven six)
3. 32 767 (thirty-two thousand seven hundred and seven)

Either we decide to go with 1 or 3. I can't find any examples in the documentation to us leaning towards either. Which of these do you prefer?

Reply:
Should we go now with 1 or 3 @jcarolinares? I like 3 more than 1, but 1 can potentially have more users (USA).

Reply:
Added this to the Style Guide, thanks guys!